src/main/java/group1/mips_simulator/components/cpuParts/Register.java

```diff
@@ -5,9 +5,15 @@
 /**
  * A Register is a space that holds a value, and makes it available for
  * the CPU to read/ write quickly.
+ *
+ * All Registers are initialized with a Value of 0, and can be modified from there
  */
 public class Register {
     public Value value = new Value(0);
+    //todo
+    //i just realized i never wrote a constructor...but Register seems to work without one
+    //since the only instance attribute gets initialized at declaration...so it doesnt need one(?)
+    // should i write one, just to be explicit?
```

Comment:
Correct. Every class object has a default constructor that thats in 0 arguments (and recursively initializes all it's fields with `null` unless specified).

You may add an explicit constructor if you wish. I think I have a different branch that gives each register a `int bitWidth = WORD_SIZE;` field so I had to make a few constructors to allow for backwards compatibility and new field initialization.